sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/Messages.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We have this here because there are error messages and a corresponding ".properties" resource file to map to. There should be that file made available.